lib/index.js

```diff
@@ -169,11 +169,17 @@ class Configurator {
 			}
 		};
 
-		if (this.opts.include) {
-			this.opts.include.forEach(
-				path => babelConfig.include.push(new RegExp(path))
-			);
-		}
+		// TODO: these modules need to be transpiled
+		// NOTE: these are npm modules, babel has a freak out if you just include all of /node_modules/
+		const es6Modules = [
+			'@financial-times/n-card',
+			'@financial-times/n-image',
+			'@financial-times/n-section',
+			'@financial-times/n-ui'
```

Comment:
I'm not sure these are necessarily default - next-article for example doesn't use n-card nor n-section. Would it make better sense to move this to be an option of nWebpack? i.e.:

```
nWebpack({
  es6Modules: [...]
  ...
});
```